test/unit/puppetlabs/services/jruby/jruby_pool_test.clj

```diff
@@ -1,211 +0,0 @@
-(ns puppetlabs.services.jruby.jruby-pool-test
-  (:import (clojure.lang ExceptionInfo))
-  (:require [clojure.test :refer :all]
-            [puppetlabs.kitchensink.core :as ks]
-            [puppetlabs.services.jruby.jruby-testutils :as jruby-testutils]
-            [puppetlabs.services.jruby.jruby-puppet-agents :as jruby-agents]
-            [puppetlabs.services.jruby.jruby-puppet-core :as jruby-core]
-            [puppetlabs.services.jruby.jruby-puppet-internal :as jruby-internal]
-            [puppetlabs.trapperkeeper.testutils.logging :as logutils]))
-
-(use-fixtures :each jruby-testutils/mock-pool-instance-fixture)
-
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;; Tests
-
-(deftest configuration-validation
```

Comment:
It might be worthwhile to have a test that the config stuff we've got now/that you are refactoring properly inserts the right defaults or errors when not passed some of these things.